homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_zone_control_zone.py

```diff
@@ -24,32 +32,127 @@
 
 PRESET_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_MODE_TO_PRESET_MODES.items()}
 
-TEMPERATURE_ZONECONTROL_DEVICE_INDEX = 20
```

Comment:
Please note this is a regression from #114006 (a test value that should definitely not have been commited after the last rebase)

The correct value has been included in this PR (L54)